serverlessworkflow/modules/ROOT/pages/index.adoc

```diff
@@ -298,64 +301,48 @@ xref:cloud/index.adoc[{product_name} in the Cloud]
 Learn about the options to deploy workflow applications in Kubernetes
 --
 
-[.card]
---
-[.card-title]
-xref:use-cases/orchestration-based-saga-pattern.adoc[Saga orchestration example in {product_name}]
-[.card-description]
-Learn how and when to use the SAGA pattern in your workflow projects
---
-
-[.card]
---
-[.card-title]
-xref:use-cases/timeout-showcase-example.adoc[Timeout example in {product_name}]
-[.card-description]
-Learn how and when to use timeout in your workflow projects
---
-
 [.card-section]
 == Integrations
 
 [.card]
 --
 [.card-title]
-xref:integrations/camel-routes-integration.adoc[Integrating with Camel Routes]
+xref:integrations/camel-routes-integration.adoc[]
 [.card-description]
 Learn how to use Camel Routes within your workflow application
 --
 
 [.card]
 --
 [.card-title]
-xref:integrations/custom-functions-knative.adoc[Invoking Knative services from {product_name}]
+xref:integrations/custom-functions-knative.adoc[]
 [.card-description]
-Learn how to invoke Knative services from {product_name} custom functions
+Learn how to invoke Knative Services from {product_name} custom functions
 --
 
 [.card]
 --
 [.card-title]
-xref:integrations/serverless-dashboard-with-runtime-data.adoc[Displaying workflow data in dashboards]
+xref:integrations/expose-metrics-to-prometheus.adoc[]
 [.card-description]
-Learn how to use dashboards to display the runtime data of your workflow application
+Exposing the workflow base metrics to Prometheus
 --
 
 [.card]
 --
 [.card-title]
-xref:integrations/expose-metrics-to-prometheus.adoc[Exposing workflow base metrics to Prometheus]
+xref:integrations/serverless-dashboard-with-runtime-data.adoc[]
 [.card-description]
-Exposing the workflow base metrics to Prometheus
+Learn how to use dashboards to display the runtime data of your workflow application
 --
 
 [.card-section]
-== Supporting Services
+== Job Services
```

Comment:
`Job Service` or `Job Services` ?
What should it be ? We only have on service available AFAIK

Reply:
It's Job Service, I think. Singular. I'll change it.